### PR TITLE
Fix Issue #1075: Unable to read/write defaultColWidth attribute in <sheetFormatPr> node

### PR DESCRIPTION
**Pull request by kigh-ota**
Added `defaultColWidth` as an optional property of `SheetFormatPropertiesModel`.

#1075

### CODE REVIEW

**Comment by Siemienik** [comment]
@kigh-ota something went wrong during testing

File: README_zh.md [hunk 1]
```diff
@@ -274,6 +274,7 @@ worksheet.properties.defaultRowHeight = 15;
 | outlineLevelCol  | 0          | 工作表列大纲级别 |
 | outlineLevelRow  | 0          | 工作表行大纲级别 |
 | defaultRowHeight | 15         | 默认行高 |
+| defaultColWidth  | (optional) | TBD |
```
Comment (Alanscut) [inline README_zh.md]:
```suggestion
| defaultColWidth  | (可选) | 默认列宽 |
```

**Comment by kigh-ota** [comment]
> @kigh-ota something went wrong during testing

@Siemienik 
Rebasing to the current master resolved the problem. Could you review again?

File: README_zh.md (hunk 1 above)
Comment (Siemienik) [reply README_zh.md]: I have no permission to commit it, @kigh-ota  could You?

Comment (kigh-ota) [reply README_zh.md]: @Alanscut 
Thank you for you transalation!

@Siemienik 
Of course. I commited the same change.
(Maybe I should have turned on the GitHub's "Allow edits from maintainers" feature.)

**Comment by akof1314** [comment]
Default column width may be specified in two different ways: using defaultColWidth and baseColWidth (see [documentation](https://msdn.microsoft.com/en-US/library/office/documentformat.openxml.spreadsheet.sheetformatproperties.aspx)).

**Pull request merged**